code/frameworks/nextjs/src/preset.ts

```diff
@@ -72,10 +72,42 @@ export const config: StorybookConfig['previewAnnotations'] = (entry = []) => [
 // You're using a version of Nextjs prior to v10, which is unsupported by this framework.
 
 export const babel = async (baseConfig: TransformOptions): Promise<TransformOptions> => {
-  configureTypescript(baseConfig);
-  configureStyledJsxTransforms(baseConfig);
+  const configPartial = loadPartialConfig({
+    ...baseConfig,
+    filename: `${getProjectRoot()}/__fake__.js`,
```

Comment:
@valentinpalkovic what's the idea here? maybe a comment would be good

Reply:
The `loadPartialConfig` function from `@babel/core` needs a filename (it doesn't have to exist) to traverse the parent tree to figure out which babel config has to be loaded. Referencing a fake/non-existent file is enough.